okio/jvm/src/main/java/okio/Buffer.kt

```diff
@@ -46,10 +46,9 @@ import javax.crypto.spec.SecretKeySpec
  */
 class Buffer : BufferedSource, BufferedSink, Cloneable, ByteChannel {
   @JvmField internal var head: Segment? = null
-  @JvmField internal var size: Long = 0L
 
-  /** Returns the number of bytes currently in this buffer.  */
-  fun size() = size
+  @get:JvmName("size")
+  var size: Long = 0L
```

Comment:
I don’t suppose I can name the getter `size()` and also access it as a field from within Kotlin?

Closest I got was having a `@JvmField private var size_` separate from the `var size` that had a getter. Felt unsatisfying, especially referring to `size_` everywhere inside of `Buffer.kt`.

Reply:
If it was `private set` you could. Not sure about how `internal set` (which you're missing, as Egor points out) behaves.